docs/100.yaml

```diff
@@ -1,3 +1,20 @@
+# This file tracks the formalisation status of theorems from Freek Wiedijk's "Formalizing 100
+# Theorems" list at https://www.cs.ru.nl/~freek/100/
+#
+# This file, contains for each theorem in the list,
+# - an entry, labelled by its number,
+# - |title|: a human-readable title
+# - (optional) |decl|: if a theorem is formalised in mathlib, the archive or counterexamples,
+#     the name of the corresponding declaration
+# - (optional) |decls|: like |decl|, but a list of declarations
+#     (if one theorem is split into multiple declarations)
+# - (optional) |doc_decls|: ...
```

Comment:
`decl` and `decls` are supposed to be mutually exclusive. You *could* mention this as well. (Or even try to check it, in yaml-check.) The same applies to 1000.yaml.